addons/analytic/security/ir.model.access.csv

```diff
@@ -1 +1 @@
-id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
Useless change

Ok, the file has to be deleted but that's out of the scope of this commit.